### PR TITLE
Homework 01 - Zahra Amalia Syarifah

### PR DESCRIPTION
**Pull request by zahraamalias**
Pull request for Homework 01 by Zahra Amalia Syarifah.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
 # hw01
 Repository for homework 1
+
+Text added through RStudio
+
+A brief biography as a part of the homework:
+
+## Zahra Amalia Syarifah
+I was born in Indonesia and grew up there. However, shortly before my high school graduation I moved to the UK. *So I **never** actually completed high school*. I went straight to a college in Southeast England and moved to Scotland for my undergraduate. Now, I am completing my Master in the States.
```
Comment (smixon) [inline README.md:9]: The use of * to emphasize bold or italics is not exactly right. It begins correctly with "So I never..." but it doesn't go to the end of the intended command. I'm not sure how to fix this, but you could try using the _ command for italics and the * for bold so there is no confusion in the line of code.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+For the sake of using the list. Here are the list of my favourite carbohydrates:
+* Pasta
+* Potato
+* Bread
+
+Here is an image of Hungarian Goulash (for the sake of the points, but also because it's one of my favourite dishes):
+
+![Image of Hungarian Goulash](http://img.taste.com.au/JCSA2ji6/w720-h480-cfill-q80/taste/2016/11/goulash-soup-75934-1.jpeg)
```
Comment (smixon) [inline README.md:18]: Included a picture with the correct code. Awesome! It took me a couple of tries to get the code for this right.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+
+You might want to make it at home, so here is a recipe for it:
```
Comment (smixon) [inline README.md:20]: Included a link to an outside website. However, it looks as if the link is written as a line separate from a hyperlink. The hyperlink command is correct, so it's needless to have both in the script. This seems like a small mistake missed in the editing process.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,36 @@
...
+
+# Reporting Process
+1. Using the forked and cloned version, I loaded README.md from the 'Files'.
+2. After I made the changes I saved it.
+3. README.md appeared under the 'Git' tab.
+4. Then I clicked 'Pull'.
+5. Then I selected 'README.md' and clicked 'Commit'.
+6. After making notes and checking the changes, I clicked 'Commit' and 'Push'.
+
+## Reflections
```
Comment (smixon) [inline README.md:32]: Zahra, I had a similar experience with this homework! A lot of what I was trying to do felt very complicated, especially the process of commits and pushes. But once I figured it out, it seemed so simple. The pain and blood was definitely worth it, and, I too, hope it gets easier.

**Review by smixon — COMMENTED** [review]
Zahra, your overall structure and presentation is coherent and easy to follow. You've used at least 4 aspects of the Markdown syntax, and you've included comments about how you made changes to the README and reflected on your experience with the assignment.

**Comment by RCruzGil** [comment]
Evaluation HW-xx for Jane Doe
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     ✓      |              |            |
| **Coding strategy**         |     ✓      |              |            |
| **Presentation**            |    ✓       |              |            |
| **Achievement, creativity** |     ✓      |              |            |
| **Ease of access**          |    ✓       |              |            |

Remarks:

* You wrote a coherent introduction to yourself and the commit comments make it really easy to follow your process. I can only find a small error in the bold code you started but didn't close. All in all, this completely covered what the assignment should be in an efficient way. And yeah, the pushing and pulling process is confusing right now. Let's only hope it gets better.

**Comment by cernhofer** [comment]
Evaluation HW-01 for zahraamalias 
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |           |       x       |            |

Remarks:
* Great work! Love the list of carbs and all of the markdown elements you used.
* Make sure to read over your final, compiled product to make sure there are no errors- it looks like you included the link to the goulash recipe twice.
* In the future, you can include any progress and/or reflections in the comment on the pull request.
* Side note- it will definitely get easier! You'll be a pro at R and git by the time the quarter is over.

**Pull request closed without merging**